test/rules/src/https_everywhere_checker/rules.py

```diff
@@ -180,15 +180,69 @@ def _determineTestApplication(self):
 		return self.test_application_problems
 
 	def getCoverageProblems(self):
-		"""Verify that each rule and each exclusion has the right number of tests
-			 that applies to it. TODO: Also check that each target has the right
+		"""Verify that each target, rule and exclusion has the right number of tests
+			 that applies to it. Also check that each target has the right
 			 number of tests. In particular left-wildcard targets should have at least
 			 three tests. Right-wildcard targets should have at least ten tests.
 
 			 Returns an array of strings reporting any coverage problems if they exist,
 			 or empty list if coverage is sufficient.
 			 """
 		problems = self._determineTestApplication()
```

Comment:
Every "`problem`" gets printed twice, they are created here in `getCoverageProblems`. This happens because `problems` is a reference to `self.test_application_problems` through `self._determineTestApplication`.  Then again in `getTargetValidityProblems` we do `problems = self._determineTestApplication`, so the `problems` from `getCoverageProblems` also get reported through `getTargetValidityProblems`.

I'm not sure why `problems` is assigned like this. I'm guessing it is a mistake in either this function or `getTargetValidityProblems`. Maybe @Hainish can chime in.

Reply:
@cschanaj You can probably just change this to:
```javascript
self._determineTestApplication()
problems = []
```
that is what other tests do